.gitlab-ci.yml

```diff
@@ -795,6 +799,7 @@ kitchen_suse:
   script:
     - cd $CI_PROJECT_DIR
     - cd $DD_AGENT_TESTING_DIR
+    - rm -rf $CI_PROJECT_DIR/kitchen_logs
```

Comment:
The point of removing .kitchen was to make sure there are no leftovers there that could interfere with subsequent runs (which I'm not sure it was the case, but it is what the kitchen folks suggested). I think it's still a good idea to delete & recreate it here.